### ISSUE TITLE
FR: localizable fieldsets (display: )

### ISSUE THREAD

**Issue by ebeauchamps** [statamic/v2-hub#1014]
FR
For almost each field in a fieldset, there is a `display: ` label

I need someting like this:
```
display:
  fr: votre adresse email
  en: your email adresse
```

Then, based on each user settings for `locale: ` the displayed text should change given this locale

**Comment by peimn** [comment]
I think it will be better to have a key for translating display:
for example:
`display:
      i18n: your_email_addr
      default: Your email address

`

and for any locale you can use that key to translate the display of fieldset.

**Comment by ebeauchamps** [comment]
I don't think so because you can have ANY type of display text given the field you want to describe. So it must be on a per field basis.
